core/console/src/main/java/org/eclipse/rdf4j/console/SetParameters.java

```diff
@@ -65,6 +90,29 @@ else if ("queryprefix".equalsIgnoreCase(key)) {
 		}
 	}
 
+	private void setLog(final String value) {
+		// Assume Logback
+		Logger logbackRootLogger = (Logger)LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+
+		if (value == null) {
+			Level currentLevel = logbackRootLogger.getLevel();
+
+			String levelString = LOG_LEVELS.inverse().getOrDefault(currentLevel, currentLevel.levelStr);
+
+			consoleIO.writeln("log: " + levelString);
```

Comment:
In the case where they successfully set the logger below, the log level should be written out also?